docs/en/start/trace/skywalking.md

```diff
@@ -0,0 +1,66 @@
+# Skywalking trace 
+
+## Configuration
+
+Example: configs/config_trace_skywalking.json
+
+````json
+{
+  "tracing": {
+    "enable": true,
+    "driver": "SkyWalking",
+    "config": {
+      "reporter": "gRPC",
+      "backend_service": "127.0.0.1:11800",
+      "service_name": "layotto"
+    }
+  }
+}
+````
+
+| Field            | Required fields | Description  |
+|------------------|-----|--------------------------|
+| reporter         | Y   | Reporting method grpc               |
+| backend_service  | Y   | skywalking oap server address |
+| service_name     | Y   | Service Name                     |
+
+## Run skywalking
+
+```shell
+cd ${project_path}/diagnostics/skywalking
+
+docker-compose -f skywalking-docker-compose.yaml up -d
+```
+
+## Run layotto
+
+```shell @if.not.exist layott
+cd ${project_path}/cmd/layotto_multiple_api/
+# build it
+go build -o layotto
+```
```

Comment:
I suggest:
~~~markdown
```shell 
cd ${project_path}/cmd/layotto_multiple_api/
```

Build it:
```shell @if.not.exist layotto
go build -o layotto
```
~~~

So that we can reuse the binary file `${project_path}/cmd/layotto_multiple_api/layotto`